docs/content/installation.rst

```diff
@@ -18,7 +18,7 @@ Python dependencies:
 * pyBigWig >= 0.3.16
 * hicmatrix >= 15
 * pysam >= 0.14
-* matplotlib >=3.1.1,<=3.3.2
+* matplotlib >=3.1.1,<=3.4.3
```

Comment:
You define in the test matrix a matplotlib version 3.5.1, but restrict it here to 3.4.3?

Reply:
done, thanks